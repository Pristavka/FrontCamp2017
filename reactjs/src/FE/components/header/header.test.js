import React from 'react';
import Header from './header';

it('Should render a Header', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});

