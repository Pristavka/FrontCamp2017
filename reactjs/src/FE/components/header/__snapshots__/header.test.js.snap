// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Header 1`] = `
<header
  className="header"
>
  <Menu />
</header>
`;
